### PR TITLE
Prepare for 3.0 final.

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

**Comment by aomarks** [comment]
> There will be a travis update that will be needed soon

What should I do?

**Comment by e111077** [comment]
Re-add the format travis check

File: demo/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2/webcomponents-loader.js"></script>
```
Comment (keanulee) [inline demo/index.html]: Didn't think we'd be updating demos to use wc-loader.js instead of -bundle.js

Comment (e111077) [reply demo/index.html]: I think loader is better as a demo should be what people should be doing irl, no?

Comment (justinfagnani) [reply demo/index.html]: Did we settle on `../node_modules/...`?

Comment (aomarks) [reply demo/index.html]: That's what we'd eventually like to have, but it depends on https://github.com/unpkg/unpkg.com/issues/74 to work through webcomponents.org, and we didn't hear back about the timing there yet. This approach of fully qualifying the URL will work locally and on webcomponents.org, so it seemed like the best way to unblock for now. We can always update them later if things change.

File: README.md [hunk 2]
```diff
@@ -41,20 +49,19 @@ your custom element project.
...
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-  <script src="/bower_components/webcomponentsjs/webcomponents-loader.js"></script>
-  <link rel="import" href="/bower_components/iron-component-page/iron-component-page.html">
-</head>
-<body>
-  <iron-component-page></iron-component-page>
-</body>
+  <head>
+    <script type="module">
```
Comment (e111077) [inline README.md]: Would it make sense to put a script to the polyfill based on this element's usage?

Comment (e111077) [reply README.md]: /shrug

**Pull request merged**